test/functional/repository/basic-methods/repository-basic-methods.ts

```diff
@@ -23,6 +23,7 @@ import { RelationAsPrimaryKey } from "./entity/RelationAsPrimaryKey"
 import { TwoUniqueColumnsEntity } from "./entity/TwoUniqueColumns"
 import { OneToOneRelationEntity } from "./entity/OneToOneRelation"
 import { UpsertOptions } from "../../../../src/repository/UpsertOptions"
+import { expect } from "chai"
```

Comment:
```suggestion
```
`import "../../../utils/test-setup"` already imports chai as a whole.

Reply:
I've undone this change. It broke other db tests that for some reason complained chai was not defined.
I looked around the rest of the code base and every file that used `expect` imports chai directly so I don't think this is a problem?